### PR TITLE
WIP: ECS capabilities

### PR DESCRIPTION
**Pull request by devinsba**
This PR is a work in progress but as I am a Java developer I thought it would be best if I start the review early.

- [x] Action: Stop ECS task
- [x] Probe: Verify ECS service is in steady state, no deployments happening
- [x] Tests

### CODE REVIEW

**Comment by Lawouach** [comment]
I'm curious Briand, are you using a linter? 

You may want to run `pycodestyle --first chaosaws` just to see if what it says. I'm curious regarding the "one line per arg". It's not usual in Python land (usually, you go as far as the expected line lentgh (79 in our projects)) and go on the next line indeed.

**Comment by devinsba** [comment]
I'm not explicitly running a linter, that command does not return any results so there is that

If you have any suggestions I am open to them. I am editing in PyCharm if that helps

**Comment by Lawouach** [comment]
If there is no outputs, then it means you pass the linting indeed.

I'm actually happy with the code as it looks :)

File: chaosaws/ecs/probes.py [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+    """
+    Checks to make sure there is not an in progress deployment
+    """
+    client = aws_client("ecs", configuration, secrets)
+    response = client.describe_services(
+        cluster=cluster,
+        services=[service]
+    )
+
+    return len(response['services']['deployments']) == 1
```
Comment (Lawouach) [inline chaosaws/ecs/probes.py]: Is there a possibility one of those two keys do not appear in the response? If that is the case, this code will break I think.

Comment (devinsba) [reply chaosaws/ecs/probes.py]: Yes, if the request results in an error, the `deployments` key will not exist. I'll fix that

Comment (devinsba) [reply chaosaws/ecs/probes.py]: This is fixed in the latest commit

File: chaosaws/ecs/probes.py [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
-__all__ = []
+from chaoslib.exceptions import FailedActivity
+from chaoslib.types import Configuration, Secrets
+
+from chaosaws import aws_client
+
+__all__ = ["service_is_not_deploying"]
+
+
+def service_is_not_deploying(cluster: str,
```
Comment (devinsba) [inline chaosaws/ecs/probes.py]: I'm wondering if this method name should be reversed. I have use cases where I want the service deploying, and I have them where I don't.

**Comment by Lawouach** [comment]
LGTM!

**Pull request merged**